ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -21,7 +21,23 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-ci-tests-kube-burner
+- as: ocp-qe-perfscale-aws-ci-tests-medium-scale-kube-burner
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      SIZE_VARIANT: large
+      WORKER_REPLICA_COUNT: "120"
```

Comment:
On Self managed clusters, you can use these env variables if you want to scale and set node type during install itself.
      COMPUTE_NODE_REPLICAS: "3"
      COMPUTE_NODE_TYPE: m5.2xlarge